woocommerce/woocommerce-android
# Removes simple payments from the orders list

Closes: #7842


### Description

The PR removes the Simple Payments short cut from the orders list. At the moment majority of the users have learned that simple payments available via "more" menu

### Testing instructions

* Open orders list
* Click on "plus"
* Notice that it goes directly to the order creation flow

Notice text change in the banner regarding "create orders" feature

### Images/gif

https://user-images.githubusercontent.com/4923871/201656128-59ed9ca0-1214-495b-81da-faeb26cea421.mp4

<img width="366" alt="image" src="https://user-images.githubusercontent.com/4923871/201844650-23879da8-601b-4d65-8259-1d8a6dfc8ce0.png">
